packages/shared/src/lib/func.ts

```diff
@@ -199,3 +199,17 @@ export const isMobile = (): boolean =>
 
 export const shouldUseNativeShare = (): boolean =>
   'share' in globalThis?.navigator && isMobile();
+
+interface BroadcastMessage {
+  eventKey: string;
+  [key: string]: unknown;
+}
+
+export const broadcastMessage = (
+  message: BroadcastMessage,
+  channelName: string = BROADCAST_CHANNEL_NAME,
+): void => {
+  const channel = new BroadcastChannel(channelName);
+  channel.postMessage(message);
```

Comment:
I don't think broadcast channel works cross origin, so everything needs to happen on the same domain